homeassistant/components/rflink.py

```diff
@@ -402,6 +406,40 @@ def async_turn_off(self, **kwargs):
         """Turn the device off."""
         return self._async_handle_command("turn_off")
 
+class CoverableRflinkDevice(RflinkCommand):
+    """Rflink entity which can switch on/stop/off (eg: cover)."""
+
+    def _handle_event(self, event):
+        """Adjust state if Rflink picks up a remote command for this device."""
+        self.cancel_queued_send_commands()
+
+        command = event['command']
+        if command in ['on', 'allon']:
+            self._state = True
+        elif command in ['off', 'alloff']:
+            self._state = False
+
+    @property
+    def should_poll(self):
```

Comment:
Is this required specifically for cover devices or is this copied over from rfxtrx? I think it might be dropped.